src/main/java/duke/Printer.java

```diff
@@ -0,0 +1,102 @@
+package duke;
+
+public class Printer {
+    /**
+     * prints out greeting from Toto upon execution
+     */
+    public static void printGreeting() {
+        String logo = "      /-\\    /-\\ \n" //6 spaces
+                + "     /  |_9_/  |\n" //5 spaces
+                + "    /,,o  3  o,,\\ \n"; //4 spaces
+        printDivider();
+        System.out.println("\tHullo I'm Toto!\n" + logo);
```

Comment:
is this a typo?

Reply:
did you mean the "hullo" ? if so, then nope 
![image](https://user-images.githubusercontent.com/57038559/93664412-5b7fc400-faa1-11ea-8402-e30010a88d87.png)

Reply:
ok